app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+  has_many :posts, dependent: :destroy
+
+  def recent_posts(id)
+    Post.where(user_id: id).last(3)
+  end
+end
```

Comment:
- [ ] It will be great to just return the most recent posts. Since you already have the association set up, you don't really need to query the post again. Hint instead of `Post.where(user_id: id).last(3)` just do `posts.order('created_at Desc').limit(3)`